src/platform/Tizen/BLEManagerImpl.cpp

```diff
@@ -954,7 +962,7 @@ CHIP_ERROR BLEManagerImpl::_Init()
 
     mServiceMode = ConnectivityManager::kCHIPoBLEServiceMode_Enabled;
 
-    ChipLogProgress(DeviceLayer, "Initialize BLE");
+    ChipLogProgress(DeviceLayer, "Initialize BLE. Tizen Version: %.1f", SystemInfo::GetPlatformVersion());
```

Comment:
I would add that to `PlatformManagerImpl::_InitChipStack()` so the log will be always visible.

Reply:
I applied it. Thank you!